minifi_main/MiNiFiWindowsService.h

```diff
@@ -20,10 +20,12 @@
 #ifdef WIN32
 
 #include <memory>
+#include <utility>
+
 #include "core/Core.h"
 
 void RunAsServiceIfNeeded();
-HANDLE GetTerminationEventHandle(bool* isStartedByService);
+std::tuple<bool, HANDLE> GetTerminationEventHandle();
```

Comment:
I think a trivial aggregate struct would be better, since it would allow naming the parts.

Reply:
we only call this function from one place, and the parts are named where we call it, so I'm not sure it's worth it

did you mean something like this? https://github.com/fgerlits/nifi-minifi-cpp/commit/02a2b692c4d924dfbb522ac0eb71a007c84dbd69

Reply:
or I guess we could do just this: https://github.com/fgerlits/nifi-minifi-cpp/commit/c81662e94abefbea9ac993a2cdb551d16ee55ecd -- still not sure it's worth it

Reply:
You can still use structured bindings at the call site.

Reply:
OK, done in c81662e94abefbea9ac993a2cdb551d16ee55ecd